chapter1/OSGPProperties.md

```diff
@@ -0,0 +1,31 @@
+## I.1 Properties of the Open Smart Grid Platform
+
+- Acts as a **connecting link** between (web)applications and smart devices
+- The open Source approach **prevents vendor lock-in** 
+- State of the art **security** 
```

Comment:
The open smart grid platform itself is security. Security devices are needed to make a secure chain (@Jonasvdbo Agree?)